hudi-utilities/src/main/java/org/apache/hudi/utilities/keygen/TimestampBasedKeyGenerator.java

```diff
@@ -39,6 +38,9 @@
 
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.apache.hudi.DataSourceUtils.checkRequiredProperties;
```

Comment:
Is this change expected ? Have the styling changed to have the static methods be imported instead of the class ?

Reply:
same as above.